src/devices/MAX31856/README.md

```diff
@@ -0,0 +1,51 @@
+﻿# MAX31856
+
+## Summary
+The MAX31856 device is a SPI interface cold-junction compensated thermocouple to digital converter.
+
+## Sensor Image
+![](device.jpg)
+
+## Usage
+The MAX31856.samples file contains a sample usage of the device. Note that this reads two temperatures. One is a connected thermocouple reading called ```temp``` and the other is the temperature of the device itself called ```tempCJ``` which is used internally to increase the accuracy of the thermocouple but can also be read if you find a use for it.
+
+Create a new ```SpiConnectionSettings``` Class if using a raspberry pi3 do not change these settings.
+
+```csharp
+SpiConnectionSettings settings = new(0, 0)
+{
+    ClockFrequency = MAX31856.SpiClockFrequency,
+    Mode = MAX31856.SpiMode,
+    DataFlow = 0
+};
+```
+
+Create a new ```SpiDevice``` with the settings from above. Then create a new MAX31856 device with the ```SpiDevice``` as well as the correct ```ThermocoupleType``` (see note below)
+```csharp
+using SpiDevice device = SpiDevice.Create(settings);
+using MAX31856 sensor = new(device, ThermocoupleType.K);
+```
+
+Now read the temperature from the device. Using the UnitsNet nuget you can convert from DegreesCelsius to DegreesFahrenheit or any other unit by changing ```.GetTemperature().DegreesFahrenheit``` to another unit of your choice.
+
+```csharp
+while (true)
+{
+    // read temperature
+    var temp = sensor.GetTemperature().DegreesFahrenheit;
```

Comment:
As the GetTemperature raises exception, you should use a Try Catch. And that's the reason why it's better to use the TryGetTemperature pattern